magstock/__init__.py

```diff
@@ -57,7 +65,7 @@ def food_cost(self):
 
     @property
     def auto_food(self):
-        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE]
+        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE] or hasattr(self, "auto_food_extra") and self.auto_food_extra
```

Comment:
Hm you know what, yea you're right.  I was remember the (unrelated) band agreement template relationship backwards.  I'll switch this.